src/components/Alert/Alerts.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './Alert.module.scss';
+
+interface Props {
+  type: string;
+}
+
+export function Alerts ({type}: Props) {
+
+    const [show, setShow] = useState(true);
+
+  if (type.match(`warning`)) {
+    type = 'warning';
+
+  }
+  else if (type.match(`info`)) {
+    type = 'info';
+  }
+  else
+    type = 'notice';
```

Comment:
This entire section of code does nothing. If the type were equal to any of these checks then they are assigned to the same value. This whole section can be removed.